src/Monitoring/Sdk/GrafanaSerialization.cs

```diff
@@ -54,7 +54,7 @@ public static IEnumerable<string> ExtractDataSourceNames(JObject dashboard)
             // datasource. This is an unsupported configuration.
 
             return dashboard
-                .SelectTokens("$..datasource")
+                .SelectTokens("$..datasource.uid")
```

Comment:
Will you please confirm that `annotation` data sources are still captured correctly? When we looked at the version 33 schema, the annotations definition in a dashboard was still the "old way".